src/unix.c

```diff
@@ -227,39 +253,23 @@ kbgetc()
     return(*kbp++);
 }
 
-#ifdef TERMIO
-
 /*
  * Set a timeout on standard input. 0 means no timeout.
  *
- * This depends on raw_state having been properly initialized, which
- * should have been done by sys_startv().
  */
```

Comment:
On reflection, I think it would be cleaner and simpler to drop the input_timeout() function in favor of setting current_timeout inside of inch().